Daramola98/WEconnectApp
# Integrate and Configure React Environment

#### What does this PR do?
Sets up react workspace using webpack

#### Description of Task to be completed?
Setup React workspace using webpack
Add JSDOC comments to all functions

#### How should this be manually tested?
1. Clone repository 
2. Navigate to the repository on your terminal
3. Change directory to client i.e cd client
4. Run command npm start
5. Webpack should compile successfully and open a page in your browser

#### Related Pivotal Tracker Stories
#155674402
